pyproject.toml

```diff
@@ -14,7 +14,8 @@ requests = "^2.24.0"
 python-dotenv = "^0.15.0"
 pymongo = "^3.11.2"
 boto3 = "^1.16.61"
-dmpy = {git = "https://github.com/ideafast/dmpy", rev = "0.1.1"}
+mypy-boto3-s3 = "^1.17.22"
```

Comment:
@jawrainey Don't this need to be a `dev` dependency? As it is only used for type checking?

Reply:
NVM - as you say, it's needed since we declare types with this package.